qdrant-landing/content/articles/what-is-quantization.md

```diff
@@ -0,0 +1,249 @@
+
+
+<img src="/articles_data/what-is-vector-quantization/hnsw-search.png" alt="HNSW Search visualization" width="300">
+
+[to add gif]
+
+And because vectors need to be stored in **fast storage** like **RAM** or **SSD** for low-latency searches, as the size of the data grows, so does the cost of storing and processing it efficiently.
+
+Quantization offers a solution by compressing vectors to smaller memory sizes, making the process more efficient. 
+
+There are several methods to achieve this, and here we will focus on three main ones:
+
+
+![Types of Quantization: 1. Scalar Quantization, 2. Product Quantization, 3. Binary Quantization](/articles_data/what-is-vector-quantization/types-of-quantization.png)
+
+
+## 1. What is Scalar Quantization?
+
+![](/articles_data/what-is-vector-quantization/astronaut-mars.jpg)
+
+In Qdrant, each dimension is represented by a float32 value, which uses **4 bytes** of memory. When using [Scalar Quantization](https://qdrant.tech/documentation/guides/quantization/#scalar-quantization), we are mapping our vectors to a range that the smaller int8 type can represent. An int8 can store 256 values (from -128 to 127, or 0 to 255) which uses only **1 byte.** This results in a **75% reduction** in memory size.
+
+For example, if our data lies in the identified range of -1.0 to 1.0, Scalar Quantization will transform these values to a range that int8 can represent, that is, within -128 to 127. So, the system **maps** the float32 values into this range.
+
+Here's a simple linear example of what this process looks like:
+
+![Scalar Quantization example](/articles_data/what-is-vector-quantization/scalar-quantization.png)
+
+To set up Scalar Quantization in Qdrant, you need to include the `quantization_config` section when creating or updating a collection:
+
+```python 
+collection_config = {
+    "vectors": {
+        "size": 128,
+        "distance": "Cosine"
+    },
+    "quantization_config": {
+        "scalar": {
+            "type": "int8",
+            "quantile": 0.99,  # Ignoring extreme outliers
+            "always_ram": True
+        }
+    }
+}
+```
+
+The **quantile** is used to calculate the quantization bounds. For example, if you specify 0.99 as the quantile, 1% of extreme values will be excluded from the quantization bounds. 
+
+This parameter only affects the resulting precision, not the memory footprint. You can tune it if you experience a significant decrease in search quality.
+
+The primary benefit of Scalar Quantization is **memory reduction.** It is especially useful for large-scale datasets, where memory and processing power become limiting factors. It also slighly improves performance, as distance calculations (such as dot product or cosine similarity) using int8 values are computationally simpler than using float32 values. 
+
+However, these performance gains are significantly lower compared to Binary Quantization, which we'll discuss later.
+
+# 2. What is Product Quantization?
```

Comment:
I strongly recommend telling first about BQ and then about PQ. PQ isn't good technique in practice and it's important to tell about BQ while the reader is not tired and doesn't stop reading

Reply:
In my personal opinion, we can only mention that PQ exists, attach the link to documentation, and don't spend users' time and energy for this. But I don't think my colleagues will agree

Reply:
I think the key is informing the user how it works and why it might not always be the best option, just like we know about it. The article consistently highlights the drawbacks.

Reply:
I've adjusted the sequence